kafka/include/userver/kafka/topic.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
```

Comment:
Maybe rename the file? `offset_range.hpp`?

Reply:
done